lib/rack/test/cookie_jar.rb

```diff
@@ -28,7 +28,7 @@ def initialize(raw, uri = nil, default_host = DEFAULT_HOST)
         @raw, options = raw.split(/[;,] */n, 2)
 
         @name, @value = parse_query(@raw, ';').to_a.first
-        @options = parse_query(options, ';')
+        @options = parse_query(options, ';').transform_keys(&:downcase)
```

Comment:
rack-test supports back to Ruby 2.0, so you'll have to change this to use a backwards-compatible implementation.

Reply:
I think I introduced a correct fix but I'm currently installing 2.0.0p0 to confirm

Reply:
I have not managed to install 2.0.0, I'm now going to hope CI runs for it